fw/procfs.c

```diff
@@ -176,17 +232,14 @@ tfw_srvstats_seq_show(struct seq_file *seq, void *off)
 	TfwSrvConn *srv_conn;
 	TfwServer *srv = seq->private;
 	unsigned int *qsize;
-	bool hm = test_bit(TFW_SRV_B_HMONITOR, &srv->flags);
-	unsigned int val[ARRAY_SIZE(tfw_pstats_ith)] = { 0 };
-	TfwPrcntlStats pstats = {
-		.ith = tfw_pstats_ith,
-		.val = val,
-	};
+	unsigned int val[T_PSZ] = { 0 };
+	TfwPrcntlStats pstats = {.val = val};
+	TfwHMStats *hm_stats;
 
 	if (!(qsize = kmalloc(sizeof(int) * srv->conn_n, GFP_KERNEL)))
```

Comment:
Same here, regarding the memory allocation - it's a very small chunk, it's short-lived, and it's happens quite regularly. Memory pool candidate?